apps/docs/src/app/examples/cards/Selectable.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import {
+  HvCard,
+  HvCardContent,
+  HvCardHeader,
+  HvStatusIcon,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  const [selected, setSelected] = useState(false);
+
+  return (
+    <div className="flex">
+      <HvCard
+        bgcolor="bgContainer"
+        className="w-full rounded-round"
+        selectable
+        selected={selected}
+      >
+        <button
+          type="button"
+          onClick={() => setSelected(!selected)}
+          aria-label="Press enter or space to select the Card."
+          className="w-full text-left"
```

Comment:
this has no hover styles, and styling is off
(nextra/tailwind must be overriding default `button` styles)
<img width="635" height="163" alt="image" src="https://github.com/user-attachments/assets/73561706-29bd-4e3d-ab76-f60302af5b5e" />

We can use our `HvButtonBase` instead, which handles `button` style reset & hover styles

```suggestion
        <HvButtonBase
          onClick={() => setSelected(!selected)}
          aria-label="Press enter or space to select the Card."
          className="flex-col text-left"
```